### ISSUE TITLE
Rename mixin to helm2

### ISSUE THREAD

**Issue by carolynvs** [getporter/helm-mixin#81]
Especially now that helm v2 is being deprecated, we should rename the mixin to make it clear which version it is for and disambiguate it from Mohamed's helm3 mixin. Once we have it renamed, let's update the [mixins list](https://porter.sh/mixins) to clearly call out both mixins.

**Comment by carolynvs** [comment]
@vdice @MChorfa  Let me know if you think this is a good idea or not. I'd like to knock this out soon if we all agree, since helm 2 is ... ☠️

**Comment by vdice** [comment]
👍 From me.  Good call.

**Comment by MChorfa** [comment]
I would agree, but this means people would change all theirs porter def? Especially old deployments

**Comment by carolynvs** [comment]
I am suggesting that we stop publishing this mixin as "helm" (leaving the old mixin available for people to download who had it scripted). But going forward we publish changes to this mixin as "helm2", and delist "helm" so that people just see "helm2" and "helm3" mixins when looking for available mixins.

Does that sound right?

**Comment by MChorfa** [comment]
Yes absolutely. It is a good plan 🙌

**Comment by carolynvs** [comment]
Great! I'll submit a PR to day to rename the mixin and announce the change on our mailing list. After helm2 mixin is live, I'll remove the old one from our search listings so people will realize they should all be using yours now.
